gammapy/modeling/models/cube.py

```diff
@@ -648,6 +648,7 @@ def __init__(
         spectral_model=None,
         spatial_model=None,
         covariance_data=None,
+        name=None,
     ):
         # TODO: remove this in v2.0
```

Comment:
Should we not do that now?

Reply:
you mean ? # TODO: remove this in v2.0
Usually we do that before the release.